tests/sources/json/test_aardvark.py

```diff
@@ -255,11 +259,11 @@ def test_aardvark_get_rights_success(aardvark_record_all_fields):
 
 def test_aardvark_get_subjects_success(aardvark_record_all_fields):
     assert MITAardvark.get_subjects(next(aardvark_record_all_fields)) == [
-        timdex.Subject(value=["Country"], kind="DCAT Keyword"),
-        timdex.Subject(value=["Political boundaries"], kind="DCAT Theme"),
-        timdex.Subject(value=["Some city, Some country"], kind="Dublin Core Spatial"),
-        timdex.Subject(value=["Geography"], kind="Dublin Core Subject"),
-        timdex.Subject(value=["Earth"], kind="Dublin Core Subject"),
+        timdex.Subject(value=["Country"], kind="DCAT; Keyword"),
+        timdex.Subject(value=["Political boundaries"], kind="DCAT; Theme"),
+        timdex.Subject(value=["Some city, Some country"], kind="Dublin Core; Spatial"),
+        timdex.Subject(value=["Geography"], kind="Dublin Core; Subject"),
+        timdex.Subject(value=["Earth"], kind="Dublin Core; Subject"),
```

Comment:
I'm digging it.  I've seen semicolons used for dilineating namespaces in freetext values like this.  Feels like a decent middle ground.